robotpy_ext/control/toggle.py

```diff
@@ -65,14 +66,15 @@ def get(self):
             else:
                 return False
 
-    def __init__(self, joystick, button, debouncePeriod=None):
+    def __init__(self, joystick: wpilib.Joystick, button: int, debounce_period=None):
```

Comment:
Should there be a type hint on `debounce_period` as well?

Reply:
indeed